client/api/assessmentStatus.js

```diff
@@ -0,0 +1,13 @@
+import request from 'superagent'
+
+export default function getAssessmentStatus () {
+  return request.get('/students/:id')
```

Comment:
this should be /status/students/${actual id for student}
this id will be passed in to getassessmentStatus(here)

Reply:
This has now been addressed. 

#closes 18